src/main.rs

```diff
@@ -22,7 +25,7 @@ mod callbacks;
 mod lua;
 
 /// Callback to route wlc logs into env_logger
-extern "C" fn log_handler(level: LogType, message_ptr: *const libc::c_char) {
+extern "C" fn log_handler(level: LogType, message_ptr: *const i8) {
```

Comment:
Because that would mean adding in the libc dependency (which we do indirectly depend on). It's also no different - the libc definition is `pub type c_char = i8;` - if this type was different from libc's, the log handler and `rustwlc::pointer_to_string` would both have compiler errors.

I know there's no real _cost_ in using libc and fixing this, but given that it's code that we'll never touch again and we have 0 other lines that could use libc I'd rather keep it off of our dependency list. If we ever need any other definition from it, I will gladly add it and replace this.